docs/Chipyard-Basics/Initial-Repo-Setup.rst

```diff
@@ -6,28 +6,41 @@ Requirements
 
 Chipyard is developed and tested on Linux-based systems.
 
-.. Warning:: It is possible to use this on macOS or other BSD-based systems, although GNU tools will need to be installed; it is also recommended to install the RISC-V toolchain from ``brew``.
+.. Warning:: It is possible to use this on macOS or other BSD-based systems, although GNU tools will need to be installed;
+    it is also recommended to install the RISC-V toolchain from ``brew``.
 
 .. Warning:: Working under Windows is not recommended.
 
+Running on AWS EC2 with FireSim
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-In CentOS-based platforms, we recommend installing the following dependencies:
+If you plan on using Chipyard alongside FireSim, you should refer to the :fsim_doc:`FireSim documentation <>`.
+Specifically, you should follow the :fsim_doc:`Initial Setup/Simulation <Initial-Setup/index.html>`
+section of the docs up until :fsim_doc:`Setting up the FireSim Repo <Initial-Setup/Setting-up-your-Manager-Instance.html#setting-up-the-firesim-repo>`.
+At that point, instead of checking out FireSim you can checkout Chipyard by following :ref:`Chipyard-Basics/Initial-Repo-Setup:Setting up the Chipyard Repo`.
 
-.. include:: /../scripts/centos-req.sh
-   :code: bash
+Default Requirements Installation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-In Ubuntu/Debian-based platforms (Ubuntu), we recommend installing the following dependencies.
-These dependencies were written based on Ubuntu 16.04 LTS and 18.04 LTS - If they don't work for you, you can try out the Docker image (:ref:`Chipyard-Basics/Initial-Repo-Setup:Pre-built Docker Image`) before manually installing or removing dependencies:
+In Chipyard, we use the `Conda <https://docs.conda.io/en/latest/>`__ package manager to help manage system dependencies.
+Conda allows users to create an "environment" that holds system dependencies like ``make``, ``gcc``, etc.
 
-.. include:: /../scripts/ubuntu-req.sh
-   :code: bash
+First Chipyard requires there to be Conda installed on the system.
+Please refer to the `Conda installation instructions <https://docs.conda.io/projects/conda/en/latest/user-guide/install/index.html>`__ on how to install Conda.
```

Comment:
If people install conda from one of the Anaconda Inc installers (miniconda or the full Anaconda), you will be configured to install packages from the 'defaults' channels built into conda (that come from Anaconda Inc).

'defaults' and 'conda-forge' are known to conflict in some gnarly ways and that's why `conda-forge` folks reccomend settings documented at https://conda-forge.org/docs/user/introduction.html#how-can-i-install-packages-from-conda-forge 

For more information on the types of conflicts that happen, see https://conda-forge.org/docs/user/tipsandtricks.html#multiple-channels

Alternately, you can avoid the whole issue by installing from one of the miniforge installers https://github.com/conda-forge/miniforge/#download which don't use anything built by Anaconda Inc and don't have the 'defaults' channel available.

Since you're using `conda env create -f scripts/conda-requirements.yml`, I _think_ the `channels` list in the YAML overrides any channels configured.  I'll go double check that I'm remembering that correctly.

Reply:
Interesting, I'll try to update the docs and point to the miniforge installers since that's probably the best thing to do.